src/video/index.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { View, Text } from "react-native";
+
+export default function VideoScreen({ webViewRef, handleVideoOn }) {
```

Comment:
export default 맨 밑으로 놓는걸로 통일하겠습니다!
```suggestion
function VideoScreen({ webViewRef, handleVideoOn }) {
```

Reply:
폴더명과 파일명을 component 이름이랑 같게 수정 부탁드립니다.  

VideoScreen.js 으로 바꿔주면 됩니다.

Reply:
네, 통일하겠습니다!